pkg/kinflate/resource/appresource_test.go

```diff
@@ -0,0 +1,63 @@
+package resource
+
+import (
+	"testing"
+
+	"reflect"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/kubectl/pkg/loader"
+)
+
+var encoded = []byte(`apiVersion: v1
+kind: Deployment
+metadata:
+  name: dply1
+---
+apiVersion: v1
+kind: Deployment
+metadata:
+  name: dply2
+`)
+
+type fakeLoader struct {
```

Comment:
Yeah, we should have a `k8s.io/kubectl/pkg/loader/loadertest`. Let's keep this for now.